src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,113 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.DukeInvalidStorageException;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+
+public class Storage {
+    private final String filePath;
+
+    /**
+     * Constructor for storage class.
+     * Creates a new file with the specified filePath if it does not exist
+     * 
+     * @param filePath filePath to store data
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+        try {
+            File f = new File(filePath);
+            f.getParentFile().mkdir();
+            f.createNewFile();
+        } catch (IOException ioException) {
+            ioException.printStackTrace();
+        }
+    }
+    
+    /**
+     * Loads the tasks that have been stored in the data file
+     * 
+     * @return List of tasks
+     */
+    public ArrayList<Task> load() {
```

Comment:
Perhaps more description might be better? Eg. loadTasks :)

Reply:
Thanks for the suggestion! Just edited the name, makes a lot more sense now :)